tests/example_pkg/setup.py

```diff
@@ -1,6 +1,6 @@
 import setuptools
 
-from unasync import bleach_build_py
+from unasync import build_py
```

Comment:
Now that the function is only named "build_py" you want to only import unasync and use `unasync.build_py'